spalp.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
```

Comment:
we'll need to update the README to remove the reference to the hook implementation

Reply:
Updated README